sormas-api/src/main/java/de/symeda/sormas/api/sormastosormas/SormasToSormasRuntimeException.java

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
I somehow feel that we already have something like this, let me see

Reply:
I've checked the RuntimeExceptions , we do not have till now a specific exception created for S2S which extends RuntimeException. If you want to use one of the custom RuntimeExceptions( maybe one with a more general naming) let me know.

Reply:
done